drivers/modem/Kconfig

```diff
@@ -39,6 +39,33 @@ config MODEM_RECEIVER_MAX_CONTEXTS
 	  Maximum number of modem receiver contexts to handle.  For most
 	  purposes this should stay at 1.
 
+config MODEM_CONTEXT
```

Comment:
I suggest marking this as experimental, so that it is easier to modify the APIs etc if we find out that there are changes needed, so just add "[EXPERIMENTAL]" at the end of bool line below.

Reply:
Good idea!  Thanks.